frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx

```diff
@@ -7,8 +7,8 @@ import { TemplateModel } from '@console/internal/models';
 import { TemplateKind } from '@console/internal/module/k8s';
 
 export const VMTemplateLink: React.FC<VMTemplateLinkProps> = ({ template }) => {
-  const name = getName(template);
-  const namespace = getNamespace(template);
+  const name = (template && template.name) || getName(template);
+  const namespace = (template && template.namespace) || getNamespace(template);
```

Comment:
```suggestion
  const namespace = template && (template.namespace || getNamespace(template));
```

makes more sense IMO

Reply:
Ahh ... yes ... sorry ...

Thanks :exclamation: fixing :-)

Reply:
done